hw/ip/clkmgr/dv/cov/clkmgr_cov_unr_excl.el

```diff
@@ -0,0 +1,154 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Generated UNR file from Synopsys UNR tool with security modules being
+// black-boxed.
+
+//==================================================
+// This file contains the Excluded objects
+// Generated By User: maturana
+// Format Version: 2
+// Date: Thu Oct  6 16:19:29 2022
+// ExclMode: default
+//==================================================
+CHECKSUM: "1607649888 3304441462"
+INSTANCE: tb.dut.u_reg.u_io_meas_ctrl_en_cdc.u_arb
+ANNOTATION: "VC_COV_UNR"
+Condition 5 "593451913" "(((!gen_wr_req.dst_req)) && gen_wr_req.dst_lat_d) 1 -1" (1 "01")
```

Comment:
i really can't tell which combination of values they think is unreachable here...

Reply:
ah okay i think i finally figured it out from the coverage report. This line to line 35 are okay.